iXBRLViewerPlugin/__init__.py

```diff
@@ -62,6 +60,15 @@ def iXBRLViewerCommandLineOptionExtender(parser, *args, **kwargs):
     # Force logging to use a buffer so that messages are retained and can be
     # retrieved for inclusion with the viewer.
     parser.add_option("--logToBuffer", action="store_true", dest="logToBuffer", default=True, help=argparse.SUPPRESS)
+    parser.add_option("--use-stub-viewer",
+                      action="store_true",
+                      dest="useStubViewer",
+                      help="Use stub viewer for faster loading of inspector")
```

Comment:
> Presumably the reason it's not on by default is due to security. 

Actually, it's just because you can no longer just open the resulting file and have it work.  Adding the note above is a good idea - now done.

> So to be sure, this doesn't reduce the delay for a user to interact with the document, it just causes the "loading" notification to appear more quickly, right?

Pretty much.  I think there is a small win from avoiding the "reparent" step, but it's primarily to get the loading mask up before rather than after the initial document load happens, which can be many seconds on a large document.